src/features/return/components/ReturnItemCard/ReturnItemCard.css

```
@@ -38,3 +38,62 @@
   grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
   gap: 16px;
 }
+
+/* 카드 */
+.item-card {
+  width: 100%;
+  min-width: 0;
+
+  border-radius: 16px;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)